docs-ui/less/sentry.less

```diff
@@ -4,14 +4,3 @@ body {
   padding: 24px;
   background: transparent;
 }
-
-html {
-  box-sizing: border-box;
-  /* font-size: 14px; */
-}
-
-*,
-*:before,
-*:after {
-  box-sizing: inherit;
-}
```

Comment:
This is now set in the `base` styles. Should probably remove this whole stylesheet but for now I’m leaving it.